modals-notifications/es5/src/js/parent.js

```diff
@@ -0,0 +1,26 @@
+function Parent(config) {
+	this.config = config;
+}
+
+Parent.prototype = {
+	constructor: Parent,
+
+	show: function () {
+		if(this.parent) {
+			this.parent.insertAdjacentHTML('afterbegin', this.html);
```

Comment:
The method show shouldn't be mutating layouts. You need all the mutations to be put to constructor